workers/unity/Packages/io.improbable.gdk.gameobjectcreation/GameObjectInitializationSystem.cs

```diff
@@ -87,9 +93,29 @@ protected override void OnDestroy()
 
         protected override void OnUpdate()
         {
-            Entities.With(newEntitiesQuery).ForEach((Entity entity, ref SpatialEntityId spatialEntityId) =>
+            Entities.With(newEntitiesQuery).ForEach((Entity entity, ref SpatialEntityId spatialEntityId, ref Metadata.Component metadata) =>
             {
-                gameObjectCreator.OnEntityCreated(new SpatialOSEntity(entity, EntityManager), Linker);
+                var spatialOsEntity = new SpatialOSEntity(entity, EntityManager);
```

Comment:
```suggestion
                var spatialOSEntity = new SpatialOSEntity(entity, EntityManager);
```

:D

Reply:
Booooooooooooooooooooo

Also this broke the PR, just revert it @paulbalaji  😛